pkg/stream/producer_daemon.go

```diff
@@ -28,6 +29,7 @@ type AggregateMarshaller func(body interface{}, attributes ...map[string]interfa
 type ProducerDaemonSettings struct {
 	Enabled           bool                   `cfg:"enabled" default:"false"`
 	Interval          time.Duration          `cfg:"interval" default:"1m"`
+	CancelDelay       time.Duration          `cfg:"cancel_delay" default:"1s"`
```

Comment:
i'm wondering if we would want to change the structure of the config here.. something like:

```
stream:
  producer:
    my-producer:
      daemon:
        enabled: true
        exec:
          cancel_delay: 1s
```

To make it more clear, that we're configuring something in the `exec` space..  
In the end this would mean, that we should rename our other `backoff` conig keys sometime in the future, too.

Reply:
Actually I think I will just remove that again... there is already a cancel delay on the output (provided by the stream package), that should be enough. Not sure I follow you on the renaming of `backoff` to `exec`. Having `backoff` as a child key of `exec` would make more sense to me there as it describes a single executor out of the `exec` package which is easy to find... Although one could think about which fields should be at `exec` and which at `exec.backoff` then.

Reply:
okay, we don't need to have this discussion now, if you remove the cancel_delay here..